contracts/PinningManager.sol

```diff
@@ -191,16 +184,11 @@ contract PinningManager {
             uint256 toTransfer = (request.numberOfPeriodsDeposited - request.numberOfPeriodsWithdrawn).mul(request.chosenPrice);
             request.numberOfPeriodsWithdrawn = 0;
             request.startDate = 0;
-            /*
-            ALLOWING_OVERFLOW reasoning: the worst that can happen is that the provider has less storage on offer than planned.
-            Verifying during all contract calls for no overflow is too costly to prevent such a small damage.
-            Damage can be easily re-done by calling setStorageCapacity. REF_CAPACITY
-            */
-            offerRegistry[msg.sender].capacity += request.size;
             provider.transfer(toTransfer); //TODO: transfer is not best practice: https://diligence.consensys.net/blog/2019/09/stop-using-soliditys-transfer-now/
             emit EarningsWithdrawn(requestReference);
         } else {
             request.size = size;
+            offer.capacity = uint128(offer.capacity.sub(size));
```

Comment:
Can we please incorporate emitting `CapacitySet` event as we need to propagate the change of the StorageOffer's capacity to Caching service as well?

Reply:
Hi Adam,
We spoke about this on Slack and I had some time to think about it. 
The answer is no, it can't be. `CapacitySet` is explicitly for the StorageProvider, who set's a capacity. To know the total available capacity of the storageProvider, you must take into account his full capacity (via `CapacitySet` event) and deduct from that all active requests (via `RequestMade`).